gcp/main.tf

```diff
@@ -37,7 +37,7 @@ locals {
 
 module "admin-server" {
   source        = "./admin-server"
-  project       = local.project
+  project       = google_project.project_n.id
```

Comment:
should we just put an explicit depends_on here to add back the dependency on the compute engine service that existed before?

Reply:
`depends_on` is to be avoided. That's one of the reasons I removed the previous dependency spec.

Reply:
How should we capture the dependency between the compute engine service and the admin server module then? This seems like an appropriate use case, although I suppose it might be good to add a comment to explain it too: https://developer.hashicorp.com/terraform/language/meta-arguments/depends_on

Reply:
I guess a little explantation is warranted: one of the advantages of terraform is that it knows better than we do the complexities of infra interdependencies. So what you want to do is hang them together naturally--i.e.: make sure that upstream dependencies are fed outputs in a natural way--instead of forcing it via `depends_on`.